docs/source/physical_robot_installation_guide/index.rst

```diff
@@ -0,0 +1,97 @@
+===============================
+Setting up a new Physical Robot
+===============================
+This tutorial is aimed towards developers, who aim to set up a new robot.
+If you simply want to know how to run a physical robot, check the examples.
+
+-----------
+Preparation
+-----------
+Firstly make sure to have all materials necessary to build the physical robot.
+Models to print the Modules can be found on the following git repo: `<https://github.com/ci-group/revolve-models>`_.
+Other items needed:
+
+* Rasberry Pi (rpi)
+* SD-card for the rpi-OS
+* suitable Battery
+* HAT for the rpi
+* cables
+
+------------------
+Setting up the RPI
+------------------
+To setup the RPI follow the steps below carefully.
+
+#. Flash the SD-card with the RPI-OS. Some Important notes:
+    * Make sure to select a appropriate Keyboard Layout (preferably English(US)).
+
+    * When setting up the WIFI connection, select your Country.
+
+#. Once the SD-card is flashed, put it into the rpi, mounted on the robot and connect to it. If you dont know the IP-address do the following:
+    * :code:`ip a` to find your ip in the **same** network.
+
+    * :code:`sudo nmap -sP <your ip>` to find all clients on your network.
+
+    * Find your rpi and connect via ssh.
+
+#. Setting up Revolve2 on the robot requires different steps, depending on the hardware version. Some general steps for all versions:
+    * In order to install Revolve2 you either have to create a venv or use a global pyenv, since rpi doesnt allow to pip install packages globally.
+        * Make and activate a venv on the rpi: :code:`python -m venv .venv` & :code:`source .venv/bin/activate`
+
+        * For :code:`pyenv` follow the steps below:
+            #. Install pyenv: :code:`curl https://pyenv.run | bash` & :code:`git clone https://github.com/pyenv/pyenv-virtualenv.git $(pyenv root)/plugins/pyenv-virtualenv`
+
+            #. Add :code:`pyenv` to bash:
+                .. code-block:: bash
+
+                    echo '# Enable pyenv
+                    export PYENV_ROOT="$HOME/.pyenv"
+                    command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
+                    eval "$(pyenv init -)"
+                    eval "$(pyenv virtualenv-init -)"
+                    ' >> ~/.bashrc
+
+            #. Log in and out of the rpi.
+
+            #. Get the right python version (this takes a long time): :code:`pyenv install 3.11`
+
+            #. create a global virtualenv: :code:`pyenv virtualenv 3.11 global_env` & :code:`pyenv global global_env`
+
+    * Clone the Revolve2 branch or repository for your installation.
```

Comment:
We do not clone and directly pip install from github instead to prevent mistakes

Reply:
once it's on pypi it will be even shorter